docs/application/native/guides/security/privacy-related-permissions.md

```diff
@@ -128,7 +128,105 @@ To check whether an application has permission to use a privilege, and to reques
+
+    ```
+    void
+    app_check_and_request_permissions()
+    {
+        ppm_check_result_e results[2];
+        const char* privileges [] = {"http://tizen.org/privilege/alarm.get",
+                                     "http://tizen.org/privilege/calendar.read"};
+        char* askable_privileges[2];
+        int askable_privileges_count = 0;
+
+        int ret = ppm_check_permissions(privileges, sizeof(privileges) / sizeof(privileges[0]), results);
+    ```
+
+    The results of the call is stored in the variable passed as the second parameter (for available results, see the `ppm_check_result_e` enumeration in [mobile](../../api/mobile/latest/group__CAPI__PRIVACY__PRIVILEGE__MANAGER__MODULE.html#ga41f409d8b9d4c27d41b907bdb0975f0c) and [wearable](../../api/wearable/latest/group__CAPI__PRIVACY__PRIVILEGE__MANAGER__MODULE.html#ga41f409d8b9d4c27d41b907bdb0975f0c) applications).
+
+2. React to the permissions check appropriately:
+
+   - If the result value is `PRIVACY_PRIVILEGE_MANAGER_CHECK_RESULT_ALLOW`, the application is allowed to perform operations related to the privilege. For example, the application can enable additional UI elements or functionalities.
+
+    ```
+        if (ret == PRIVACY_PRIVILEGE_MANAGER_ERROR_NONE) {
+            for (int it = 0; it < sizeof(privileges) / sizeof(privileges[0]); ++it)
+            {
+                switch (results[it]) {
+                case PRIVACY_PRIVILEGE_MANAGER_CHECK_RESULT_ALLOW:
+                    /* Update UI and start accessing protected functionality */
+                    break;
+    ```
+
+   - If the result value is `PRIVACY_PRIVILEGE_MANAGER_CHECK_RESULT_DENY`, the application is not allowed to perform operations related to the privilege. Any attempt to use such functionality without the user's consent fails. Usually, this means that invoking any API function that involves the privilege results in an error.
+
+     ```
+                case PRIVACY_PRIVILEGE_MANAGER_CHECK_RESULT_DENY:
+                    /* Show a message and terminate the application */
+                    break;
+
+     ```
+
+   - If the result value is `PRIVACY_PRIVILEGE_MANAGER_CHECK_RESULT_ASK`, the application must request permission from the user with the `ppm_request_permissions()` function, which displays a dialog box. You can pass user data as the forth parameter of the function, as a pointer to an application context instance. When the user makes a decision, a callback defined as the second parameter is invoked.
+
+     The dialog box asking for user permissions is shown only if the `ppm_request_permissions()` function returns without an error.
+
+    ```
+                case PRIVACY_PRIVILEGE_MANAGER_CHECK_RESULT_ASK:
+                   askable_privileges[askable_privileges_count++] = privileges[it];
+                   /* Log and handle errors */
+                   break;
+                }
```

Comment:
Line 185- 194, can be indented in....

Reply:
lines 182 - 184 had wrong indent.
Lines from 185 to 194 are correct.